Formula/m/mit-scheme.rb

```diff
@@ -1,10 +1,11 @@
 class MitScheme < Formula
   desc "MIT/GNU Scheme development tools and runtime library"
   homepage "https://www.gnu.org/software/mit-scheme/"
-  url "https://ftp.gnu.org/gnu/mit-scheme/stable.pkg/12.1/mit-scheme-12.1.tar.gz"
-  mirror "https://ftpmirror.gnu.org/gnu/mit-scheme/stable.pkg/12.1/mit-scheme-12.1.tar.gz"
-  sha256 "5509fb69482f671257ab4c62e63b366a918e9e04734feb9f5ac588aa19709bc6"
+  url "https://ftp.gnu.org/gnu/mit-scheme/stable.pkg/12.1/mit-scheme-12.1-svm1-64le.tar.gz"
```

Comment:
what is the diff between the two?

Reply:
The svm1 binary, as dictated on [GNU's website](https://www.gnu.org/software/mit-scheme/) is a tarball that allows `make` to compile on any UNIX system with any architecture.  

The one that I removed was the source code tarball which was "For unix systems; uses linefeeds as line delimiters." whereas the SVM1 portable binary is "For use on any 64-bit unix system (including Apple silicon).". Thus, necessitating the change. 

This change passes all checks and creates new bottles which did not exist before as the old revision of the package did not compile at all.